core/src/test/scala/org/apache/spark/sql/delta/DeltaErrorsSuite.scala

```diff
@@ -108,8 +108,8 @@ trait DeltaErrorsSuiteBase
     testUrls()
   }
 
-
-  test("test DeltaErrors OSS methods") {
+  // Disabled due to delta-io/delta/issue/1070
+  ignore("test DeltaErrors OSS methods") {
```

Comment:
this will be reenabled after fixing the other issue .. right?

Reply:
Thats correct, but this PR is for 1.2.1 branch only.